languages/amh.js

```diff
@@ -0,0 +1,210 @@
+// Keyboard Language
+// please update this section to match this language and email me with corrections!
+// "all" is used here for example purposes, by convention it should be the ISO 639-1 code for the specified language
+// ***********************
+jQuery.keyboard.language.all = {
```

Comment:
Please replace the "all" with the language abbreviation for Ethiopic... I tried to find one using [ISO-639-3](https://en.wikipedia.org/wiki/ISO_639-3) language codes, but it isn't listed.